### PR TITLE
Fixed missing sourcemaps when building assets.

### PR DESCRIPTION
**Pull request by theoshallburn**
### Ticket
- [https://jira.stibodx.com/browse/BNNCUE-4180](BNNCUE-4180)

### Description
- Gulp has built-in support for sourcemaps, using this feature works and creates correct sourcemaps. The previously used plugin did not.

### CODE REVIEW

**Review by colourgarden — APPROVED** [review]
Please bump the version number in `package.json`.

**Review by wrux — APPROVED** [review]
Tested this ins a few projects and it seems to be working without any issues 👍

**Comment by theoshallburn** [comment]
@sun can you have a look and merge this if it is fine? Two of your colleagues already gave their approval.

File: tasks/scripts.js [hunk 1]
```diff
@@ -9,18 +9,16 @@ module.exports = (gulp, $, pkg) => {
   function scriptTask(opts) {
     if (!pkg.gulpPaths.scripts.src) { return false }
     const options = getOptions($, pkg.gulpPaths.scripts.options, opts);
-    return gulp.src(pkg.gulpPaths.scripts.src)
+    return gulp.src(pkg.gulpPaths.scripts.src, {sourcemaps: options.sourcemaps})
```
Comment (sun) [inline tasks/scripts.js:12]:
```suggestion
    return gulp.src(pkg.gulpPaths.scripts.src, { sourcemaps: options.sourcemaps })
```

File: tasks/styles.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ module.exports = (gulp, $, pkg) => {
   function styleTask(opts) {
     if (!pkg.gulpPaths.styles.src) { return false }
     const options = getOptions($, pkg.gulpPaths.styles.options, opts);
-    return gulp.src(pkg.gulpPaths.styles.src)
+    return gulp.src(pkg.gulpPaths.styles.src, {sourcemaps: options.sourcemaps})
```
Comment (sun) [inline tasks/styles.js:35]:
```suggestion
    return gulp.src(pkg.gulpPaths.styles.src, { sourcemaps: options.sourcemaps })
```

File: tasks/scripts.js [hunk 3]
```diff
@@ -9,18 +9,16 @@ module.exports = (gulp, $, pkg) => {
...
       .pipe($.eslint.format())
       .pipe($.if(options['fail-after-error'], $.eslint.failAfterError()))
-      .pipe($.if(options.sourcemaps, $.sourcemaps.init()))
       .pipe($.babel({
         presets: [['@babel/preset-env', { modules: false }]],
       }))
       .pipe($.if(options.concat, $.concat(pkg.title.toLowerCase().replace(/[^a-z]/g,'') + '.js')))
-      .pipe($.if(options.sourcemaps, $.sourcemaps.write()))
-      .pipe(gulp.dest(pkg.gulpPaths.scripts.dest))
+      .pipe(gulp.dest(pkg.gulpPaths.scripts.dest, {sourcemaps: options.sourcemaps}))
```
Comment (sun) [inline tasks/scripts.js:21]:
```suggestion
      .pipe(gulp.dest(pkg.gulpPaths.scripts.dest, { sourcemaps: options.sourcemaps }))
```

File: tasks/styles.js [hunk 4]
```diff
@@ -60,7 +58,7 @@ module.exports = (gulp, $, pkg) => {
         },
         targetFileType: ['jpe?g', 'png', 'webp', 'svg', 'gif', 'ico', 'otf', 'ttf', 'eot', 'woff2?'],
       }))
-      .pipe(gulp.dest(pkg.gulpPaths.styles.dest))
+      .pipe(gulp.dest(pkg.gulpPaths.styles.dest, {sourcemaps: options.sourcemaps}))
```
Comment (sun) [inline tasks/styles.js:61]:
```suggestion
      .pipe(gulp.dest(pkg.gulpPaths.styles.dest, { sourcemaps: options.sourcemaps }))
```

**Review by sun — APPROVED** [review]

**Pull request merged**